chsdi/views/features.py

```diff
@@ -269,6 +267,10 @@ def _identify_grid(params, layerBodIds):
         grid = Grid(gridSpec.get('extent'),
                     gridSpec.get('resolutionX'),
                     gridSpec.get('resolutionY'))
+        if params.srid == 2056 and gridSpec.get('srid') == '21781':
```

Comment:
@procrastinatio works for GeoJson Grids in LV95 now. But not yet in webmercator... and even now my elif dows look ugly. Any idea?

Reply:
ping @procrastinatio